Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -127,7 +127,7 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
       .filter(featureNativeCheckoutPledgeViewIsEnabled >>> isFalse)
 
     let shouldGoToManagePledge = ctaButtonTappedWithType
-      .filter { $0 == .viewBacking || $0 == .manage }
+      .filter(shouldGoToManagePledge(with:))
```

Comment:
The compiler didn't let me simply add `$0 == .fix` here, so I had to extract the logic to a separated function.